Automattic/wp-calypso
# Domain Redirects: Release to all users

Related to pb5gDS-3gV-p2

## Proposed Changes

* Releasing Domain Redirects to all users
* Removing the support link (temporarily)

## Testing Instructions



* Test the Domain Redirects feature without flags and ensure everything works as expected: http://calypso.localhost:3000/domains/manage/all/{YOUR_DOMAIN}/edit/{YOUR_DOMAIN}


## Pre-merge Checklist
